docs/hello_nextflow/09_hello_nf-core.md

```diff
@@ -244,4 +244,1050 @@ You have a general idea of what nf-core offers and you know how to run an nf-cor
+        - `Include citations`
+        - `Include a gitpod environment`
+        - `Include GitHub Codespaces`
+        - `Use fastqc`
+        - `Add a changelog`
+        - `Support Microsoft Teams notifications`
+        - `Support Slack notifications`
+    6. Select **Finish** on the Final details screen
+    7. Wait for the pipeline to be created, then select **Continue**
+    8. Select **Finish without creating a repo** on the Create GitHub repository screen
+    9. Select **Close** on the HowTo create a GitHub repository page
+
+If run successfully, you will see a new folder in your current directory named `myorg-myfirstpipeline`.
+
+###  Testing your pipeline
+
+Let's try it:
+
+```bash
+cd /workspace/gitpod/hello-nextflow/hello-nf-core/myorg-myfirstpipeline
+nextflow run . -profile docker,test --outdir results
+```
+
+The pipeline should run successfully!
+
+Here's the console output from the pipeline:
+
+```console title="Output"
+Launching `./main.nf` [marvelous_saha] DSL2 - revision: a633aedb88
+
+Input/output options
+  input                     : https://raw.githubusercontent.com/nf-core/test-datasets/viralrecon/samplesheet/samplesheet_test_illumina_amplicon.csv
+  outdir                    : results
+
+Institutional config options
+  config_profile_name       : Test profile
+  config_profile_description: Minimal test dataset to check pipeline function
+
+Core Nextflow options
+  runName                   : marvelous_saha
+  containerEngine           : docker
+  launchDir                 : /workspace/gitpod/hello-nextflow/hello-nf-core/myorg-myfirstpipeline
+  workDir                   : /workspace/gitpod/hello-nextflow/hello-nf-core/myorg-myfirstpipeline/work
+  projectDir                : /workspace/gitpod/hello-nextflow/hello-nf-core/myorg-myfirstpipeline
+  userName                  : gitpod
+  profile                   : docker,test
+  configFiles               : 
+
+!! Only displaying parameters that differ from the pipeline defaults !!
+------------------------------------------------------
+executor >  local (1)
+[ba/579181] process > MYORG_MYFIRSTPIPELINE:MYFIRSTPIPELINE:MULTIQC [100%] 1 of 1 ✔
+-[myorg/myfirstpipeline] Pipeline completed successfully-
+```
+
+Let's dissect what we are seeing:
+
+The nf-core pipeline template is a working pipeline and comes pre-configured with some modules, here we only chose MultiQC:
+
+-   [MultiQC](https://multiqc.info/): A modular tool to aggregate results from bioinformatics analyses across many samples into a single report.
+
+!!! note "The template can be granularly configured"
+
+    From nf-core tools 3.0 onwards many features can be removed during template creation. This is what we did when deselecting 
+    features earlier.
+
+At the top you see, all parameters displayed that differ from the pipeline defaults. These were configured with the `test` profile.
+
+You can use the `test` profile to check if your pipeline is still working during your development cycle. 
```

Comment:
I find short stop-start sentences a bit hard to follow. Join them up a bit into a narrative?